main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapViewModel.java

```diff
@@ -20,6 +20,8 @@
 import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.ViewModel;
 
+import javax.annotation.Nullable;
```

Comment:
IIRC we use `androidx.annotation.Nullable` normally. Not sure what's the difference thought... (but it'll get obsolete anyway if we remove the setter again)

Reply:
Yeah, don't know why I've select this one from the suggestions (or whether it got added automatically). Anyway, is removed meanwhile.